### DISCUSSION TITLE
CNAME

### DISCUSSION THREAD

**Discussion by Niranjanreddypemmaka** [vercel/vercel#4222] [General]
How can i add CNAME to my domain(it is a thired party domain like godaddy ) in versel .

**Comment by coetry** [comment]
If your domain is hosted with another provider like GoDaddy, you can set a CNAME record for any subdomain you wish and set the value as `alias.zeit.co` (will be `cname.vercel-dns.com` on [May 20th](https://www.notion.so/vercel/Change-Timeline-fab7622780694753a92c0a396cea7eb6))

Then you can add that subdomain to any of your projects as described [here](https://vercel.com/docs/v2/custom-domains#subdomains).

If your nameservers are pointed to Vercel, there is no configuration necessary and you can just add any subdomain to your projects.

If your nameservers are pointed to Vercel, but you would like to point to an external address, then you can use the [CLI](https://vercel.com/docs/now-cli#commands/dns) to add that record.

Let me know if I answered your question!
